### ISSUE TITLE
LOGO Particle: Wrong Link

### ISSUE THREAD

**Issue by Omegatcu** [gantry/gantry5#57]
Hello - i just added my own Logo in the Particle and I think this particle has a little bug ;)

Here the Code that is generated:

```html
<div class="g-content">
<a href="/images/LogoMetall.png" title="DOWN-Gilde" rel="home" class="gantry-logo">
    <img src="/" alt="DOWN-Gilde">
</a>
</div>
```

As you see the img src is empty but the link is not - should be the other way around.

**Comment by w00fz** [comment]
I tried to replicate this but seems to work fine. Can you share how you set up the particle settings in the admin? I think this was fixed already in c0f57af28d69.

**Comment by Omegatcu** [comment]
your right - I tested it in a new install of Beta 2 and its now working the right way. 
Sorry it was a beta 1 Site and I installed the framework and Template again as Beta 2 but it seems it didn't take all of the changes. - Reinstalling Beta2 was the Fix.
